ext/nnstreamer/extra/nnstreamer_protobuf.cc

```diff
@@ -149,7 +149,8 @@ gst_tensor_converter_protobuf (GstBuffer *in_buf, GstTensorsConfig *config, void
 
   for (guint i = 0; i < config->info.num_tensors; i++) {
     const nnstreamer::protobuf::Tensor *tensor = &tensors.tensor (i);
-    const gchar *name = tensor->name ().c_str ();
+    std::string _name = tensor->name ();
+    const gchar *name = _name.c_str ();
 
     config->info.info[i].name = (name && strlen (name) > 0) ? g_strdup (name) : NULL;
```

Comment:
```suggestion
    const gchar *name = g_strdup (tensor->name ().c_str ());

    config->info.info[i].name = name;
```

Is it wrong?

Reply:
In this case, even if `tensor->name ()` is "", `name` is not null.
In the test, check whether the name is null pointer (NULL).

Reply:
As we discussed via chat,  a string whose length is 0 is "\0", not NULL. We need to consider it here and in the related unit tests.

Reply:
In the process of converting to a stream buffers, NULL was temporarily replaced with "".
So it's converted back to NULL.